avae/data.py

```diff
@@ -39,6 +39,12 @@ def load_data(
         print("\nData size:", len(data))
         print("\nClass list:", data.final_classes)
 
+        plot_affinity_matrix(
+            lookup=lookup,
+            all_classes=lookup.columns.tolist(),
+            selected_classes=data.final_classes,
+        )
+
```

Comment:
I don't think I can actually checkout and run this because this is a fork (correct me if I am wrong and there's a way to do this), but I *think* this will fail because ```lookup``` can be ```None``` at this stage (line 29). I suggest moving this function call onto line 27 inside the if statement after the lookup is actually read.